form.yml

```diff
@@ -7,7 +7,20 @@ form:
   - imagefile
   - custom_num_cores
 attributes:
-  imagefile: "scipy-notebook_lab-4.0.7.sif"
+  imagefile:
+    display: true
+    widget: "select"
```

Comment:
Yes, scipy- and datascience- are both standard images, built from jupyter's repo without customization. The other ones are course specific, although `torch` is used by two different classes taught by Logan, which is why I just called it "Torch".

I think we might have the ability to change image visibility with some use of Ruby syntax and setting `form.yml` to be `form.yml.erb` instead. I took a quick look and didn't see an obvious way to have Ruby pull the groups for the user, or for a context variable that might have something useful. I think it will require some better understanding of what one can do in Ruby, if it's possible at all. There's [docs from Open OnDemand](https://osc.github.io/ood-documentation/latest/how-tos/app-development/interactive/form.html), but I didn't see anything that outlines all of the variables that are available in the form specifically, and I wasn't able to get that to print out in a tooltip by modifying the .erb file, but I don't think I plumbed the depths of what's possible.